package.json

```diff
@@ -88,6 +88,8 @@
     "fela-plugin-rtl": "^1.0.6",
     "keyboard-key": "^1.0.1",
     "lodash": "^4.17.10",
+    "mobx": "^5.1.2",
```

Comment:
another point to rather use React context for achieving the same goal is that currently we have to include mobx to the list of dependencies - while, essentially, this is used only for docs experience (not something that **lib** consumer is interested in)